using-vertx/src/main/java/org/acme/vertx/PgClientProducer.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2019 Red Hat, Inc.
+ *
+ * Red Hat licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.acme.vertx;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Produces;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import io.reactiverse.axle.pgclient.PgClient;
+import io.reactiverse.axle.pgclient.PgPool;
+import io.reactiverse.pgclient.PgPoolOptions;
+import io.vertx.axle.core.Vertx;
+
+@ApplicationScoped
+public class PgClientProducer {
+
+    @Inject
+    @ConfigProperty(name = "pg.pool.database", defaultValue = "quarkus_test")
+    String database;
+
+    @Inject
+    @ConfigProperty(name = "pg.pool.user", defaultValue = "quarkus_test")
+    String user;
+
+    @Inject
+    @ConfigProperty(name = "pg.pool.password", defaultValue = "quarkus_test")
+    String password;
+
+    @Inject
+    @ConfigProperty(name = "pg.schema.create", defaultValue = "true")
+    boolean schemaCreate;
+
+    @Inject
+    Vertx vertx;
+
+    @Produces
+    PgPool client;
+
+    @PostConstruct
+    void config() {
+        PgPoolOptions options = new PgPoolOptions()
+                .setDatabase(database)
+                .setUser(user)
+                .setPassword(password);
+
+        client = PgClient.pool(vertx, options);
+
+        if (schemaCreate) {
+            initdb();
+        }
+    }
+
+    private void initdb() {
+        client.query("DROP TABLE IF EXISTS fruits")
```

Comment:
Can't it reuse the import.sql support?

Reply:
Yeah, Panache RX is going to try to offer that too. ATM it's ORM that's doing the import, so there's nothing he can use to do that.

Reply:
We could but for what benefit? It would add complexity to this method which is used as a snippet in the guide to introduce the `query` client operation.

Reply:
No, I think what @cescoffier meant was that if you had a file named `import.sql` in your resource folder, then quarkus should take care of doing that insertion for you.

Reply:
@FroMage there's no `import.sql` in this quickstart for now. Tell me what's expected because your last comment confused me.

Reply:
Yes, I'm saying in quarkus for Hibernate ORM we put these statements in an `import.sql` file that is automagically executed at startup. We should probably do the same for the pg-client quarkus extension.

Reply:
OK. Is it fine with you if `import.sql` contains both DML and DDD statements? In the Hibernate ORM extension it is different since they have a specific config property for drop and create schema.